modules/storages/app/controllers/storages/admin/storages_controller.rb

```diff
@@ -166,7 +166,7 @@ def default_breadcrumb
   # See: default_breadcrum above
   # Defines whether to show breadcrumbs on the page or not.
   def show_local_breadcrumb
-    true
+    !OpenProject::FeatureDecisions.storage_primer_design_active?
```

Comment:
I am wondering whether we really need the feature flag after all. 🤔 Maybe we can discuss that later.

Reply:
Since we've moved the edit & delete buttons into the storage edit/view page- removing the feature flag at this stage would be disruptive to the user as they wouldn't have a delete action 😬 My plan was to retain the flag and remove it once we have the edit page built out-